examples/BandFrag/bandfrag.py

```diff
@@ -0,0 +1,56 @@
+from scm.plams import Settings
+from scm.plams import fromASE
+from scm.plams.recipes.bandfragment import BANDFragmentJob
+
+# build the surface
+from ase.build import fcc111, add_adsorbate
+
+mol = fcc111("Au", size=(2, 2, 3))
+add_adsorbate(mol, "H", 1.5, "ontop")
+mol.center(vacuum=10.0, axis=2)
+
+# separate the fragments
+surface = mol.copy()
+symbols = surface.get_chemical_symbols()
+del surface[[i for i in range(len(symbols)) if symbols[i] != "Au"]]
+adsorbate = mol.copy()
+del adsorbate[[i for i in range(len(symbols)) if symbols[i] == "Au"]]
+
+# optional: load the optimized molecules
+from ase import io
+
+surface_opt = io.read("surface_opt.xyz")
+adsorbate_opt = io.read("adsorbate_opt.xyz")
+assert len(surface_opt) == len(surface)
+assert len(adsorbate_opt) == len(adsorbate)
```

Comment:
Should this be commented out by default in the example? If not, please include the .xyz files alongside the example.